client/my-app/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Vite + React</title>
+</head>
+
+<body>
+  <h1>Backlog for PGM4</h1>
```

Comment:
I don't think it's wrong per se but I would avoid editing index.html with actual content, though you could add meta tags or scripts or ... This `h1` would be better located in one of your components as it's actually a part (= the title) of your React application.